src/main/java/seedu/address/logic/commands/addcommands/AddStudentToGroupCommand.java

```diff
@@ -31,54 +35,62 @@ public class AddStudentToGroupCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD + "/" + COMMAND_WORD_ALIAS
         + ": Adds a student to a group identified by the group name used.\n"
         + "Parameters: "
-        + PREFIX_STUDENT_NUMBER + "STUDENT_NUMBER "
+        + "[" + PREFIX_STUDENT_NUMBER + "STUDENT_NUMBER]... "
         + PREFIX_GROUP_NAME + "GROUP_NAME\n"
         + "Example: " + COMMAND_WORD + " "
         + PREFIX_STUDENT_NUMBER + "A02345678J "
+        + PREFIX_STUDENT_NUMBER + "A0456789K "
         + PREFIX_GROUP_NAME + "CS2103T-T14-1";
 
-    public static final String MESSAGE_SUCCESS = "Added student: %1$s to %2$s";
+    public static final String MESSAGE_SUCCESS = "Added the following students to %1$s:\n%2$s";
```

Comment:
Can you follow Soo Yap's implementation [here](https://github.com/AY2425S1-CS2103-F12-2/tp/pull/287/files) under AddGroupCommand.java 

Allows user to see if they entered duplicate values